xbmc/pictures/GUIWindowSlideShow.cpp

```diff
@@ -670,9 +682,7 @@ void CGUIWindowSlideShow::Process(unsigned int currentTime, CDirtyRegionList &re
 
   RenderPause();
   if (IsVideo(*m_slides.at(m_iCurrentSlide)) && appPlayer && appPlayer->IsRenderingGuiLayer())
-  {
```

Comment:
I'd keep the braces for readability

Reply:
Done. And thanks for giving it a look :slightly_smiling_face: